### ISSUE TITLE
SQL Server Input - Add Readable SQL Server Version to ServerProperties

### ISSUE THREAD

**Issue by Trovalo** [influxdata/telegraf#8252]
## Feature Request

Add the readable version of the SQL Server Instance

### Proposal:
Add a new tag [sql_version_desc] to the measurement _sqlserver_serverproperties_.
It contains the SQL Server version in a readable way.

The value can be fetched using
```TSQL
SELECT LEFT(@@VERSION,CHARINDEX(' - ',@@VERSION)) AS [sql_version_desc]
```
The result is ```Microsoft SQL Server 2016 (SP2-GDR) (KB4532097)``` and is coherent in any SQL Server version from 2008 and later. It also works for Azure (even if is way less useful).

### Current behavior:
This data is available only in numeric format es "13.0.5102.14" in the tag "sql_version"

### Desired behavior:
Add an additional tag "sql_version_desc" which displayed the readable version, es: "Microsoft SQL Server 2016 (SP2-GDR) (KB4532097)"

**Comment by Trovalo** [comment]
I've got the PR ready for the on-prem and I'd like to know if it should be added also for azure.
@denzilribeiro do you think adding this would be useful for azure Instances?

**Comment by denzilribeiro** [comment]
Go ahead add it for on-prem , there is a PR out for Azure. 
https://github.com/influxdata/telegraf/pull/8322

**Comment by Trovalo** [comment]
from what I see the numeric [sql_version] exists only in Azure Managed Instance but is completely absent in Azure SQL DB
```
,SERVERPROPERTY('ProductVersion') AS [sql_version]
```

has it been made (not gathered) on purpose?

**Comment by denzilribeiro** [comment]
> from what I see the numeric [sql_version] exists only in Azure Managed Instance but is completely absent in Azure SQL DB
> 
> ```
> ,SERVERPROPERTY('ProductVersion') AS [sql_version]
> ```
> 
> has it been made (not gathered) on purpose?

Yes there is only 1 version for Azure SQL DB :)  We could collect it as it changes slightly but can't related it to anything else, will never have multiple versions unless in middle of deployment cycle.

**Comment by Trovalo** [comment]
I suspected that, I'll leave Azure SQL DB as it is now.
